src/components/UI/common/Footer.tsx

```diff
@@ -0,0 +1,105 @@
+import { Box, Center, Divider, Flex, Link, Stack } from '@chakra-ui/react';
+import Image from 'next/image';
+import { FC } from 'react';
+import efLogoSVG from '../../../public/images/ef-logo.svg';
+import githubLogoSVG from '../../../public/images/github-logo.svg';
+import twitterLogoSVG from '../../../public/images/twitter-logo.svg';
+import { ImportantText, PageText } from '../text';
+
+export const Footer: FC = () => {
+  return (
+    <Flex id='footer' bgGradient='linear(to-b, #E07861 10%, #E06361 100%)' d='column'>
```

Comment:
Should we use variables for these hex colors?

Reply:
Moved to `theme`